### ISSUE TITLE
original() causes a typescript error

### ISSUE THREAD

**Issue by tmtron** [immerjs/immer#505]
## Link to repro
[Stackblitz example](https://stackblitz.com/edit/typescript-immer-test?file=index.ts)

## To Reproduce

use the example code from the [immer original docs](https://github.com/immerjs/immer/blob/master/docs/original.md) with typescript

```typescript
const baseState = {users: [{name: "Richie"}]}
const nextState = produce(baseState, draftState => {
	original(draftState.users) // is === baseState.users
}) 
```

## Observed behavior

When using the `original()` function as described in the [docs](https://github.com/immerjs/immer/blob/master/docs/original.md), we get a typescript error:

```
Argument of type '{ name: string; }[]' is not assignable to parameter of type 'Drafted<{ name: string; }[], ImmerState>'.
  Property '[DRAFT_STATE]' is missing in type '{ name: string; }[]' but required in type '{ [DRAFT_STATE]: ImmerState; }'.
```

## Expected behavior

No typescript errors

## Workaround

cast to any:

```typescript
	original(draftState.users as any) // is === baseState.users
```

## Environments

* IntelliJ  with typescript `3.4.5`
* [Stackblitz example](https://stackblitz.com/edit/typescript-immer-test?file=index.ts)

- **Immer version:**
- [x] Occurs with `setUseProxies(true)`
- [x] Occurs with `setUseProxies(false)` (ES5 only)

**Comment by mweststrate** [comment]
This is a regression indeed, will be fixed in 5.3 through #506

**Comment by tmtron** [comment]
Thanks, that was fast!
just updated to 5.3.0 and can confirm that this works now.
